static/main.css

```diff
@@ -190,6 +190,23 @@ dd + dt { margin-top: 1em; }
   display: inline !important;
 }
 
+.feed-icon {
+  color: #f26522;
+}
+
+.feed-icon span {
+  height: 14px;
+  width: 100%;
+  display: inline !important;
+  padding: .2em;
+}
+
+.feed-icon:hover {
+  border-bottom: 0;
+  background-color: #f26522;
+  color: white;
+}
```

Comment:
I'm not sure if custom styling is needed. We don't need to keep the original 14x14 size, ideally it would be inline with the text. Not sure if those colours will work in both light and dark mode.